src/model/GeneralizedHiddenMarkovModel.cpp

```diff
@@ -57,7 +60,7 @@ GeneralizedHiddenMarkovModel::GeneralizedHiddenMarkovModel(
 void
 GeneralizedHiddenMarkovModel::initializeCache(CEPtr<Standard> /* evaluator */,
                                               unsigned int /* phase */) {
-  // TODO(igorbonadio)
+  throw tops::exception::NotYetImplemented(__func__);
```

Comment:
You can discard `tops::` in this context. As all models are within `namespace tops`, they can find symbols inside it (you must qualify, however, `exception`, as in `exception::NotYetImplemented`).